packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -101,8 +102,14 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
   private _initializeApp() {
     // TODO: Maybe pass this in as a parameter instead of creating it here?
     this.state.app = express()
+    Sentry.init({
+      dsn: this.options.sentryDsn,
+      tracesSampleRate: 1.0,
+    })
```

Comment:
Does this error if the `dsn` is `undefined`? Do we want tracing?

Reply:
^i'll test out behavior

tracing is easy to add so can do that! what are your thoughts on tracing one or both these services?

Reply:
I'm not against tracing, it just adds scope. If you think there is a low overhead to adding it then go for it! I think as-is it is sending `1.0` = 100% of requests through the tracing module? Surely that adds overhead? Maybe it should be something like `0.05` = 5%?

Reply:
Should probably only initialize if the env bar is set?

Reply:
undefined dsn just issues a warning, so i think we're good here

Reply:
@snario added tracing here since it's simple, will monitor and iterate on this